ansible/advanced.md

```diff
@@ -84,6 +94,20 @@ meanings for different followers.
 * ER-301 can be sent gate and CV commands, with the 4 tracks
   corresponding to the first 4 `SC` channels.
 
+* Disting Ex, left to right: allocate notes/gates polyphonically, map each
```

Comment:
ah jeez, forgive my ignorance -- Disting Ex has 5 different operating modes, yeah?

i might actually re-do this formatting upon merge to lay them out as bullets, since there's a lot of text with the commas, but just wanted to confirm i was reading this right. thank you so much, @mrmattallison !

Reply:
Hi Dan,

Apologies if it's not clear, Disting EX has 4 different operating modes.

Left to right...

1. single algorithm plays notes, voice allocated by the EX
2. single algorithm plays notes, voices are fixed
3. MIDI (via a MIDI beakout), all notes are sent on MIDI channel 1
4. MIDI, voices 1-4 sent on MIDI channels 1-4

In MIDI modes the Disting will not play single algorithms, it's used a MIDI convertor, though dual mode algorithms can be used.

Reply:
AH, yeah, the comma-separated layout doesn't allow for this kind of granularity -- this is super helpful information!! i think it's much clearer to have numbered descriptions of the different modes.

would you like to adjust? i can when its merged, but happy to give you the learning opp! :)